### PR TITLE
cleaned up errors in regards to filtering data from api calls, marker…

### PR DESCRIPTION
**Pull request by iSingh-24**
markers correctly showing up with no errors. Still need to check and see if we can filter by genre instead of keyword.

### CODE REVIEW

File: client/Components/FilterMap.jsx [hunk 1]
```diff
@@ -42,10 +61,10 @@ const Filter= ()=>{
           open={Boolean(menuOpen)}
           onClose={closeMenu}
         >
-          <MenuItem color='black' className={classes.container}>Rock</MenuItem>
-          <MenuItem className={classes.container}>Pop</MenuItem>
-          <MenuItem className={classes.container}>Jazz</MenuItem>
-          <MenuItem className={classes.container}>Country</MenuItem>
+          <MenuItem value='rock' color='black' className={classes.container} onClick={filterMapData}>Rock</MenuItem>
```
Comment (alejvillanueva) [inline client/Components/FilterMap.jsx:64]: Why does only rock have a value?

File: client/Components/FilterMap.jsx [hunk 2]
```diff
@@ -30,6 +36,19 @@ const Filter= ()=>{
         setMenuOpen(false);
     }
 
+    const filterMapData = async(event) => {
+     
+      const newVal = event.currentTarget.innerText;
```
Comment (alejvillanueva) [inline client/Components/FilterMap.jsx:41]: Does currentTarget.value work for this?

**Review by alejvillanueva — APPROVED** [review]
Just some questions, but other wise looks good!

**Review by vikkiwchan — COMMENTED** [review]
+1 @alejvillanueva 's comments

**Pull request merged**